--- sendmail-8.18.2/mailstats/mailstats.8	2023-02-09 19:24:38.000000000 +0100
+++ sendmail-8.18.2/mailstats/mailstats.8	2026-08-03 04:10:50.027492417 +0200
@@ -103,7 +103,7 @@
 .SH FILES
 .PD 0.2v
 .TP 2.5i
-/etc/mail/sendmail.cf
+/etc/sendmail.cf
 The default
 .B sendmail
 configuration file.
--- sendmail-8.18.2/sendmail/sendmail.0	2025-12-23 06:37:30.000000000 +0100
+++ sendmail-8.18.2/sendmail/sendmail.0	2026-08-03 04:10:50.028064481 +0200
@@ -431,13 +431,13 @@
        names  are  all specified in [4m/etc/mail/sendmail.cf[24m.  Thus, these values
        are only approximations.
 
-        /etc/mail/aliases
+        /etc/aliases
               raw data for alias names
 
         /etc/mail/aliases.db
               data base of alias names
 
-        /etc/mail/sendmail.cf
+        /etc/sendmail.cf
               configuration file
 
         /etc/mail/helpfile
--- sendmail-8.18.2/sendmail/sendmail.8	2024-01-24 18:09:22.000000000 +0100
+++ sendmail-8.18.2/sendmail/sendmail.8	2026-08-03 04:10:50.027780169 +0200
@@ -709,14 +709,14 @@
 .I "Sendmail Installation and Operation Guide"
 .SH FILES
 Except for the file
-.I /etc/mail/sendmail.cf
+.I /etc/sendmail.cf
 itself the following pathnames are all specified in
-.IR /etc/mail/sendmail.cf .
+.IR /etc/sendmail.cf .
 Thus,
 these values are only approximations.
 .PP
 .TP
- /etc/mail/aliases
+ /etc/aliases
 raw data for alias names
 .TP
  /etc/mail/aliases.db
--- sendmail-8.18.2/sendmail/newaliases.0	2025-12-23 06:37:30.000000000 +0100
+++ sendmail-8.18.2/sendmail/newaliases.0	2026-08-03 04:10:50.028271213 +0200
@@ -8,7 +8,7 @@
 
 [1mDESCRIPTION[0m
        [1mNewaliases  [22mrebuilds  the  random access data base for the mail aliases
-       file /etc/mail/aliases.  It must be run each time this file is  changed
+       file /etc/aliases.  It must be run each time this file is  changed
        in order for the change to take effect.
 
        [1mNewaliases [22mis identical to ``sendmail -bi''.
@@ -20,7 +20,7 @@
        [1msendmail.[0m
 
 [1mFILES[0m
-       /etc/mail/aliases   The mail aliases file
+       /etc/aliases   The mail aliases file
 
 [1mSEE ALSO[0m
        aliases(5), sendmail(8)
--- sendmail-8.18.2/sendmail/newaliases.1	2025-10-20 21:09:43.000000000 +0200
+++ sendmail-8.18.2/sendmail/newaliases.1	2026-08-03 04:10:50.028344123 +0200
@@ -20,7 +20,7 @@
 .SH DESCRIPTION
 .B Newaliases
 rebuilds the random access data base for the mail aliases file
-/etc/mail/aliases.  It must be run each time this file is changed
+/etc/aliases.  It must be run each time this file is changed
 in order for the change to take effect.
 .PP
 .B Newaliases
@@ -40,7 +40,7 @@
 .B sendmail.
 .SH FILES
 .TP 2i
-/etc/mail/aliases
+/etc/aliases
 The mail aliases file
 .SH SEE ALSO
 aliases(5), sendmail(8)
--- sendmail-8.18.2/cf/m4/cfhead.m4	2023-02-09 19:24:38.000000000 +0100
+++ sendmail-8.18.2/cf/m4/cfhead.m4	2026-08-03 04:10:50.028419564 +0200
@@ -275,7 +275,7 @@
 define(`DATABASE_MAP_TYPE', `hash')
 
 # set up default values for options
-define(`ALIAS_FILE', `MAIL_SETTINGS_DIR`'aliases')
+define(`ALIAS_FILE', `/etc/aliases')
 define(`confMAILER_NAME', ``MAILER-DAEMON'')
 define(`confFROM_LINE', `From $g $d')
 define(`confOPERATORS', `.:%@!^/[]+')
--- sendmail-8.18.2/praliases/praliases.8	2023-02-09 19:24:38.000000000 +0100
+++ sendmail-8.18.2/praliases/praliases.8	2026-08-03 04:10:50.028595445 +0200
@@ -48,7 +48,7 @@
 utility exits 0 on success, and >0 if an error occurs.
 .SH FILES
 .TP 2.5i
-/etc/mail/sendmail.cf
+/etc/sendmail.cf
 The default
 .B sendmail
 configuration file.
