--- a/cf/m4/proto.m4	2025-10-20 21:09:43.000000000 +0200
+++ b/cf/m4/proto.m4	2026-08-03 04:13:16.929118247 +0200
@@ -500,10 +500,10 @@
 _OPTION(TryNullMXList, `confTRY_NULL_MX_LIST', `False')
 
 # load average at which we just queue messages
-_OPTION(QueueLA, `confQUEUE_LA', `8')
+_OPTION(QueueLA, `confQUEUE_LA', `100')
 
 # load average at which we refuse connections
-_OPTION(RefuseLA, `confREFUSE_LA', `12')
+_OPTION(RefuseLA, `confREFUSE_LA', `100')
 
 # log interval when refusing connections for this long
 _OPTION(RejectLogInterval, `confREJECT_LOG_INTERVAL', `3h')
