--- a/sendmail/srvrsmtp.c	2025-11-26 17:08:32.000000000 +0100
+++ b/sendmail/srvrsmtp.c	2026-08-03 04:12:27.948386616 +0200
@@ -783,13 +783,13 @@
 				bool tsave = QuickAbort;		\
 									\
 				QuickAbort = false;			\
-				usrerr(response);			\
+				usrerr("%s", response);			\
 				QuickAbort = tsave;			\
 				e->e_sendqueue = NULL;			\
 				goto doquit;				\
 			}						\
 			else						\
-				usrerr(response);			\
+				usrerr("%s", response);			\
 			break;						\
 									\
 		  case SMFIR_REJECT:					\
@@ -2119,7 +2119,7 @@
 					if (ISSMTPREPLY(nullserver))
 					{
 						/* Can't use ("%s", ...) due to usrerr() requirements */
-						usrerr(nullserver);
+						usrerr("%s", nullserver);
 					}
 					else
 					{
@@ -2777,7 +2777,7 @@
 					if (response != NULL)
 					{
 						/* Can't use ("%s", ...) due to usrerr() requirements */
-						usrerr(response);
+						usrerr("%s", response);
 					}
 					else
 					{
@@ -4169,7 +4169,7 @@
 # endif
 
 			/* Can't use ("%s", ...) due to usrerr() requirements */
-			usrerr(response);
+			usrerr("%s", response);
 			if (strncmp(response, "421 ", 4) == 0
 			    || strncmp(response, "421-", 4) == 0)
 			{
@@ -4297,7 +4297,7 @@
 				(void) extenhsc(response + 4, ' ', e->e_enhsc);
 # endif
 			/* Can't use ("%s", ...) due to usrerr() requirements */
-			usrerr(response);
+			usrerr("%s", response);
 			if (strncmp(response, "421 ", 4) == 0
 			    || strncmp(response, "421-", 4) == 0)
 				rv = false;
